docs/sources/mimir/release-notes/v2.16.md

```diff
@@ -0,0 +1,83 @@
+- Experimental PromQL functions are disabled by default but can be enabled using only the per-tenant setting `enabled_promql_experimental_functions`.
+
+Support for native histograms and out-of-order native histograms is enabled by default in ingesters.
+
+Distributors discard float and histogram samples with duplicated timestamps from each timeseries in a request before the request is forwarded to ingesters.
+Discarded samples are tracked by `cortex_discarded_samples_total` metrics with the reason `sample_duplicate_timestamp`.
+
+## Experimental features
+
+Grafana Mimir 2.16 includes some features that are experimental and disabled by default.
+Use these features with caution and report any issues that you encounter:
+
+Distributors now include experimental support for the Influx [line protocol](https://docs.influxdata.com/influxdb/cloud/reference/syntax/line-protocol/).
+
+Query-frontends now include experimental support to "spin off" subqueries as actual range queries, so that they benefit from query acceleration techniques such as sharding, splitting, and caching.
+
+## Bug fixes
+
+- Distributor: Use a boolean to track changes while merging the ReplicaDesc components, rather than comparing the objects directly. #10185
+- Querier: fix timeout responding to query-frontend when response size is very close to `-querier.frontend-client.grpc-max-send-msg-size`. #10154
+- Query-frontend and querier: show warning/info annotations in some cases where they were missing (if a lazy querier was used). #10277
+- Query-frontend: Fix an issue where transient errors are inadvertently cached. #10537 #10631
+- Ruler: fix indeterminate rules being always run concurrently (instead of never) when `-ruler.max-independent-rule-evaluation-concurrency` is set. https://github.com/prometheus/prometheus/pull/15560 #10258
+- PromQL: Fix various UTF-8 bugs related to quoting. https://github.com/prometheus/prometheus/pull/15531 #10258
+- Ruler: Fixed an issue when using the experimental `-ruler.max-independent-rule-evaluation-concurrency` feature, where if a rule group was eligible for concurrency, it would flap between running concurrently or not based on the time it took after running concurrently. #9726 #10189
+- Mimirtool: `remote-read` commands will now return data. #10286
+- PromQL: Fix deriv, predict_linear and double_exponential_smoothing with histograms https://github.com/prometheus/prometheus/pull/15686 #10383
+- MQE: Fix deriv with histograms #10383
+- PromQL: Fix <aggr_over_time> functions with histograms https://github.com/prometheus/prometheus/pull/15711 #10400
+- MQE: Fix <aggr_over_time> functions with histograms #10400
+- Distributor: return HTTP status 415 Unsupported Media Type instead of 200 Success for Remote Write 2.0 until we support it. #10423 #10916
```

Comment:
```suggestion
- Distributor: Return HTTP status 415, Unsupported Media Type, instead of 200, Success, for Remote Write 2.0 until we support it. #10423 #10916
```